pyproject.toml

```diff
@@ -77,6 +77,7 @@ doc = [
     "matplotlib==3.6.3",
     "numpydoc==1.5.0",
     "pandas==1.5.3",
+    "panel==0.14.3",
```

Comment:
Why `panel` is needed ?

Reply:
I needed it locally as I use the `panel` backend [here](https://github.com/pyansys/pymapdl/pull/1828/files#:~:text=pv.set_jupyter_backend(%22panel%22)) . That's why I added it but if you think it's not necessary, you can remove it.